test.sh

```diff
@@ -8,7 +8,7 @@ flake8 .
 ./.buildifier.sh
 
 echo "Running tests..."
-bazel test --test_output=errors reconciletags:reconciletags_test
+bazel test --test_output=errors appengine/reconciletags:reconciletags_test
 bazel test --test_output=errors ftl/... --deleted_packages=ftl/node/benchmark,ftl/php/benchmark,ftl/python/benchmark,ftl/benchmark
 bazel test --test_output=errors testing/lib:mock_registry_tests
-cd runtime_builders && py.test test_manifest.py && cd ..
+cd appengine/runtime_builders && py.test test_manifest.py && cd ..
```

Comment:
Shouldn't you go back up two directories here?

Reply:
Ah yeah, I guess it didn't matter since it's at the end? Fixed it with pushd.